bridge/contracts/AllowTokens.sol

```diff
@@ -114,11 +114,11 @@ contract AllowTokens is Initializable, UpgradableOwnable, UpgradableSecondary, I
     }
 
     function _setTypeLimits(uint256 typeId, Limits memory limits) private {
-        require(typeId < typeDescriptions.length, "AllowTokens: bigger than typeDescriptions");
-        require(limits.max >= limits.min, "AllowTokens: maxTokens smaller than minTokens");
-        require(limits.daily >= limits.max, "AllowTokens: dailyLimit smaller than maxTokens");
-        require(limits.mediumAmount > limits.min, "AllowTokens: limits.mediumAmount smaller than min");
-        require(limits.largeAmount > limits.mediumAmount, "AllowTokens: limits.largeAmount smaller than mediumAmount");
+        require(typeId < typeDescriptions.length, "AllowTokens: > typeDescriptions");
+        require(limits.max >= limits.min, "AllowTokens: Tokens max < min");
+        require(limits.daily >= limits.max, "AllowTokens: daily < maxTokens");
+        require(limits.mediumAmount > limits.min, "AllowTokens: mediumAmount < min");
```

Comment:
limits.mediumAmount instead of mediumAmount? I think it might be clearer to the reader